packages/editor/src/lib/editor/managers/SnapManager/SnapManager.test.ts

```diff
@@ -330,7 +331,7 @@ describe('SnapManager', () => {
 
 			editor.getSortedChildIdsForParent.mockReturnValue([frameId])
 			editor.getShape.mockReturnValue(frameShape as any)
-			editor.isShapeOfType.mockImplementation((_shape, type) => type === 'frame')
+			editor.isShapeOfType.mockImplementation((_shape: any, type: any) => type === 'frame')
```

Comment:
There's more addition of `: any` than i'd like with the mocks. does vitest just have worse types or did we lose some specificity in the TestEditor?

Reply:
ah I guess it's the vitest `Mocked` wrapper type being less good than the jest one?